contracts/CourtPresaleActivate.sol

```diff
@@ -153,7 +161,11 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
 
         // activate in registry
         bondedToken.approve(address(registry), bondedTokensObtained);
-        registry.stakeFor(_from, bondedTokensObtained, abi.encodePacked(ACTIVATE_DATA));
+        bytes memory data;
+        if (_activate) {
+            data = abi.encodePacked(ACTIVATE_DATA);
+        }
+        registry.stakeFor(_from, bondedTokensObtained, data);
 
         // refund leftovers if any
         _refund(bondedToken);
```

Comment:
We can't see it in this PR, but `BoughtAndActivated` is a bit of a misnomer now since it's emitted even if you don't activate. If we'd like to keep this old one, maybe we can emit a different event for just `Bought`?

Reply:
Good point. Actually I was thinking of removing the event, as we already have an event in the Presale contract for the purchase and an event in the registry for the activation. I'll ask the frontend guys.

Reply:
Let's move the conversation here:
https://aragon.chat/channel/dev?msg=W45awEJaX3hGwaE9G
@Evalir @sohkai What should I do? Rename it? Remove it? Leave it as it is?

Reply:
It sounds like we still need it... so maybe we can add another event for just `Bought`?

@Evalir are we only using the event to show confirmations or do we need old events?

Reply:
We're only using `BoughtAndActivated` for setting the success messages in the converter; It would be useful for to have `Bought` and `Activation` events to closely tell the user what they actually have done when they get their ANJ.

Reply:
Would it be enough with these events?
https://github.com/aragonone/court-presale/blob/master/app/contracts/BalanceRedirectPresale.sol#L189
https://github.com/aragon/aragon-court/blob/master/contracts/registry/JurorsRegistry.sol#L670
https://github.com/aragon/aragon-court/blob/master/contracts/registry/JurorsRegistry.sol#L583

Reply:
> We're only using BoughtAndActivated for setting the success messages in the converter

In this case, we should know based on whether the user's transaction was mined successfully or not and don't need to be listening to the event.